Fix temp file names for ram_over_time

derekprior opened the pull request:
The temp file generated, at least on OS X, was invalid in some manner.
It would end up splitting into two files - one that contained the
time stamp and output from `ruby -v` and the other that contained the
rest of the file name. Neither of these files ever had any actual output
recorded to them.

None of the other logs have the ruby version in them so instead of
trying to figure out what characters OS X wasn't liking, I just removed
the ruby version from the name.

schneems commented:
I originally wrote this to explicitly test different patch versions of ruby, i.e. to see how your app performed in different ruby versions. I had it output to files since the tasks take so long to run and I would forget which was which. Is there a way we could maybe sanitize the ruby version instead?

derekprior commented:
Yup. I was just being lazy. I'll look into it - maybe tomorrow.

On Thu, Aug 20, 2015 at 4:58 PM, Richard Schneeman <notifications@github.com
> wrote:

> I originally wrote this to explicitly test different patch versions of
> ruby, i.e. to see how your app performed in different ruby versions. I had
> it output to files since the tasks take so long to run and I would forget
> which was which. Is there a way we could maybe sanitize the ruby version
> instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/schneems/derailed_benchmarks/pull/49#issuecomment-133168515>
> .
>

derekprior commented:
Looks like it was the newline at the end of the ruby version causing screwy behavior. I changed the PR to address that and updated the commit message.

schneems commented:
Ping, make any progress here?

derekprior commented:
Yes. See coment above yours (and the new diff) :smile: .

schneems commented:
Sorry missed that. Cool, thanks :heart: :rocket:
